tests/test_physical_systems/test_mechanical_loads.py

```diff
@@ -249,21 +249,21 @@ class TestExtSpeedLoad(TestMechanicalLoad):
 
     key = 'ExtSpeedLoad'
     class_to_test = ExternalSpeedLoad
-    kwargs = dict(speed_profile=test_speed_profile,
+    kwargs = dict(speed_profile=speed_profile,
```

Comment:
this is no good style and poses a potential side effect.
Functions can also be keys in a dictionary, since they're also objects.

Please use a different key id or a different function name here.

Reply:
Done.